.changes/unreleased/charts-console-Changed-20250310-162338.yaml

```diff
@@ -0,0 +1,12 @@
+project: charts/console
+kind: Changed
+body: |-
+    Prepare Console charts for Console v3 configuration changes.
+    Redpanda Console now supports unified authentication and authorization between Console and Redpanda, including user impersonation. This means you can authenticate to Redpanda using the same credentials you use for Redpanda Console.
+    `config` is now a generic top level property.
+    `roles` and `roleBindings` have been removed.
+    Secrets for schema registry have their dedicated stanza.
+    Secrets for `login` configuration has been removed and replaced with `authentication` stanza, which supports configuring OIDC.
+    Secret for `protobufGitBasicAuthPassword` has been moved to a separate `serde` stanza.
+    `license` and `licenseSecretRef` have been moved to be top level properties.
```

Comment:
This doesn't feel very consumable for end users. The target audience is someone that's installed console via the helm chart and now wants to upgrade to V3. Try to keep it short, sweet, and fully specified. When possible, link to external resources instead of duplicating information.

```
Upgrade to [Console V3](Link to Console v3 release notes) (BREAKING CHANGES)

  - `full.path.to.old.key` has been moved to `full.path.to.new.key`
  - `a.removed.value` has been removed.
  - `a.changed.value` has been replaced by [`better.path`](Link to README)
```